testsupport/wait/host.go

```diff
@@ -1023,18 +1023,67 @@ func (a *HostAwaitility) WaitForNSTemplateTierAndCheckTemplates(t *testing.T, na
 		if ns.TemplateRef == "" {
 			return nil, fmt.Errorf("missing 'templateRef' in namespace #%d in NSTemplateTier '%s'", i, tier.Name)
 		}
-		if _, err := a.WaitForTierTemplate(t, ns.TemplateRef); err != nil {
+		tierTemplateNamespaces, err := a.WaitForTierTemplate(t, ns.TemplateRef)
+		if err != nil {
 			return nil, err
 		}
+		// if the tier template supports Tier Template Revisions then let's check those
+		if tierTemplateNamespaces.Spec.TemplateObjects != nil {
+			if _, err := a.WaitForTierTemplateRevision(t, tierTemplateNamespaces.GetName(), UntilTierTemplateRevisionsHasOwnerReferences([]metav1.OwnerReference{
+				{
+					APIVersion: "toolchain.dev.openshift.com/v1alpha1", // for some reason, the apiversion and kind are empty on the CR
+					Kind:       "TierTemplate",
+					Name:       tierTemplateNamespaces.Name,
+				},
```

Comment:
let's drop this, as discussed in the other PR

Reply:
see my comment here https://github.com/codeready-toolchain/host-operator/pull/1103#discussion_r1849142054

Reply:
removed here: https://github.com/codeready-toolchain/toolchain-e2e/pull/1061/commits/9cb5461bfe8b5124bc538959dda9c296aa669d8f